.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,61 @@
+name: Docker
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+on:
+  push:
+    branches: [ main, master ]
+    # Publish semver tags as releases.
+    tags: [ 'v*.*.*' ]
+  pull_request:
+    branches: [ main, master ]
+
+env:
+  # Use docker.io for Docker Hub if empty
+  REGISTRY: ghcr.io
+  # github.repository as <account>/<repo>
+  IMAGE_NAME: ${{ github.repository }}
```

Comment:
Our image name doesn't follow this naming pattern.

Reply:
We are only using this variable once, why the indirection?

Reply:
I guess we could rename it to make things easier.

Reply:
I will remove it but the docker images need to follow the github repo naming

Reply:
Didn't know that, thanks for pointing it out!